feature_set_quickstart_guide/main/conda.yaml

```diff
@@ -0,0 +1,14 @@
+name: feature_set_quickstart_guide
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - python=3.9
+  - scipy
+  - scikit-learn
+  - pandas
+  - catboost
+  - pip:
+    - qwak-sdk # SDK for Qwak platform - latest version
```

Comment:
no need for qwak-sdk here (unless it's for local usage?)

Reply:
They test locally, but also install the qwak sdk manually so.. IDK - I actually thought that's mandatory to add it to the dependencies